src/styles/index.scss

```diff
@@ -1,3 +1,133 @@
+@import "variables";
+
 body {
   margin: 0;
 }
+
+.product {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  width: 200px;
+  height: 406px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: $white;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+  }
+
+  &__name {
+    width: 166px;
+    margin-top: 40px;
+    margin-bottom: 0;
+    color: $fiol;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    align-self: flex-start;
+    margin-left: 16px;
+    margin-top: 4px;
+    color: $gray;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__rating {
+    margin-top: 6px;
+    display: flex;
+    width: 166px;
+    justify-content: space-between;
+    align-items: flex-end;
+
+    &-stars {
+      display: flex;
+      justify-content: space-between;
+      margin: 0;
+      width: 96px;
+      height: 16px;
+    }
+
+    &-stars-star {
+      width: 16px;
+      height: 16px;
+      background-image: url(../images/star.svg);
+      background-position: center;
+      background-repeat: no-repeat;
+    }
+
+    &-stars :nth-last-child(n + 2) {
+      background-image: url(../images/star-active.svg);
+    }
```

Comment:
Remember that the stars--N modifier shoul have an affect on how many stars are active